app/assets/stylesheets/content/_choice.sass

```diff
@@ -26,10 +26,29 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
+//a little hack, the width will be just fine for this element
+$choice-parent-relation-select-width: 325px
```

Comment:
some times the button goes in a second row, I changed the width of select element.